spyderlib/plugins/editor.py

```diff
@@ -93,14 +93,14 @@ def clear_breakpoint(filename, lineno):
 class EditorConfigPage(PluginConfigPage):
     def get_name(self):
         return _("Editor")
-        

```

Comment:
Ok, first comment: please revert **all** this blank space changes :-) This makes merging other PRs really, really hard.